case-lib/opt.sh

```diff
@@ -115,27 +116,13 @@ func_opt_parse_option()
         fi
     done
     
-    [ ${OPT_VALUE_lst['h']} -eq 1 ] && _func_opt_dump_help
-    # verify & check all option values
-    [[ $(declare -p |grep "[[:space:]]OPT_VALUE_lst=\"\"") ]] && echo "Missing parameter/script default value" && _func_opt_dump_help
-    # record the full parameter to the cmd
-    if [ ! -f $LOG_ROOT/cmd.txt ]; then
-        local cmd="$0" opt
-        for opt in ${!OPT_OPT_lst[@]}
-        do
-            [[ $opt == "h" ]] && continue
-            if [ ${OPT_PARM_lst[$opt]} -eq 1 ]; then
-                cmd="$cmd -$opt ${OPT_VALUE_lst[$opt]}"
-            else
-                if [ $(echo '!'${OPT_VALUE_lst[$opt]}|bc) -eq 1 ]; then
-                    cmd="$cmd -$opt"
-                fi
-            fi
-        done
-        echo $cmd >> $LOG_ROOT/cmd.txt
-        local git_path=$(dirname $0)
-        echo "Branch: "$(git -C $git_path branch |grep '^*') >> $LOG_ROOT/version.txt
-        echo "Commit: "$(git -C $git_path log --oneline -1) >> $LOG_ROOT/version.txt
+    [ "${OPT_VALUE_lst['h']}" -eq 1 ] && _func_opt_dump_help
+    # record the git version
+    if [ ! -f "$LOG_ROOT/version.txt" ]; then
+        local git_path
+        git_path=$(dirname "$0")
```

Comment:
Can you make `git_path` a global at the top? For consistency with other scripts even if not needed for now.

Reply:
good idea, I will update and check it.